controllers/home-routes.js

```diff
@@ -1,11 +1,29 @@
+const sequelize = require("../config/connection");
 const router = require('express').Router();
-// const { User, Song, Battle } = require('../models');
+const { User, Song } = require('../models');
 
 // Handlebar homepage renders with dummy object
-router.get('/', (req,res) => {
-   return res.render('homepage', {appName:"Sputtle"})
-})
-
+router.get('/', (req, res) => {
+    Song.findAll({ 
+      order: sequelize.random(),
+      limit: 2
+    })
+    .then(dbSongData => {
+      // console.log(dbSongData)
+      const songs = dbSongData.map(song => song.get({ plain: true }))
+      const data = { 
+        id: req.session.id,
```

Comment:
an attempt of fixing my double homepage state. we might be able to keep all the data together and then log in to see the battle